### PR TITLE
Vester module enhancement

### PR DESCRIPTION
**Pull request by equelin**
- Add the possibility to provide multiple config files
- Add tests to check if the config files exists
- Add Pester tests to validate the module configuration (guid, version) and functions help

### CODE REVIEW

File: Vester/Public/Invoke-Vester.ps1 [hunk 1]
```diff
@@ -16,6 +16,18 @@ function Invoke-Vester {
     Invoke-Vester then calls Pester to run each test file. The test files
     leverage PowerCLI to gather values for comparison/remediation.
 
+    .PARAMETER Script
```
Comment (brianbunke) [inline Vester/Public/Invoke-Vester.ps1]: These aren't needed. The comments I've added directly above the actual parameters will auto-populate each `.PARAMETER` in the comment-based help.

File: Vester/Public/Invoke-Vester.ps1 [hunk 2]
```diff
@@ -74,47 +86,52 @@ function Invoke-Vester {
         # Optionally define a different config file to use
         # Defaults to Vester\Configs\Config.ps1
         # Currently only supports one Config file at a time
```
Comment (brianbunke) [inline Vester/Public/Invoke-Vester.ps1]: This third comment now obsolete with your update :+1:

**Comment by brianbunke** [comment]
Ok, first: The change to Invoke-Vester technically works well, I think, and I want to support it. But I have to bring up a couple questions before a stamp of approval, because I'm a glutton for long, boring punishment.

1. The submission, as is, behaves differently whether tests are piped in or not. Thanks, ForEach loops.
  1. Try it yourself:
    1. `Invoke-Vester -Script .\Tests\VM -Config '.Configs\Config.ps1','.Configs\Config.ps1' -Verbose`
    2. `gci .\Tests\VM | Invoke-Vester -Config '.Configs\Config.ps1','.Configs\Config.ps1' -Verbose`
  2. `a.` does all tests at a time for the first config; `b.` does all configs at a time for the first test
  3. Do we care?
2. We will be accepting multiple tests and multiple configs. Only one param will be able to accept piped input from Get-ChildItem. Do we care which one?

**Comment by equelin** [comment]
Thanks for your code review Brian! I've learned some stuff today :+1: 

Concerning your questions about pipelining Tests/Configs, I think we should fisrt answer the 2nd question and then we could talk about the first one. I personally see more value for pipelining config files rather than tests.  Config files should be our first citizens in Vester, especially when will be able to select which test to run from the config file itself.

**Comment by chriswahl** [comment]
Agree with @equelin on configurations being the primary focus. I like the Invoke-Vester -Script .\Tests\VM -Config '.Configs\Config.ps1','.Configs\Config.ps1' -Verbose example as it looks cleaner and more intuitive, too.

**Comment by brianbunke** [comment]
Regarding my tangent, we can definitely explore flipping the pipe over to -Config. Because of that, we don't have to worry about the pipe order behavior right now.

Changes made, PR looks good! :shipit:

**Pull request merged**